aztk/spark/models/plugins/install/install.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+apt-get update
```

Comment:
shouldn't that only be for the apt?

Reply:
yeah, this is going to be refactored out. I think that having a single `install.sh` is fine so long as we do a check that $COMMAND is apt-get.

Reply:
Yeah I'll move it to the full command